proto/zitadel/idp.proto

```diff
@@ -470,6 +470,8 @@ message SAMLConfig {
     // Optional name of the attribute, which will be used to map the user
     // in case the nameid-format returned is `urn:oasis:names:tc:SAML:2.0:nameid-format:transient`.
     optional string transient_mapping_attribute_name = 5;
+    // In case the metadata could not be processed, the corresponding error will be returned.
+    optional string metadata_error = 6;
```

Comment:
As mentioned already, I don't really see that we would need this attribute in the query side.